profiler/src/ProfilerEngine/Datadog.Profiler.Native/AppDomainStore.cpp

```diff
@@ -16,6 +16,16 @@ AppDomainStore::AppDomainStore(ICorProfilerInfo4* pProfilerInfo)
 
 bool AppDomainStore::GetInfo(AppDomainID appDomainId, ProcessID& pid, std::string& appDomainName)
 {
+    std::unique_lock lock{_lock};
+
+    auto it = _appDomainToInfo.find(appDomainId);
+
+    if (it != _appDomainToInfo.cend())
+    {
+        std::tie(pid, appDomainName) = it->second;
```

Comment:
Nice. But what do you need the Pid per AppDomain for? The process it's supposed to be only one

Reply:
👋 you are right, I just wanted to keep the same behavior as before.
I plan to refactor it later.

Thanks for the review